src/botPage/common/__tests__/TicksService.js

```diff
@@ -2,10 +2,11 @@ import { expect } from 'chai';
 import websocket from 'ws';
 import { LiveApi } from 'binary-live-api';
 import TicksService from '../TicksService';
+import { getAppIdFallback } from '../../../common/appId';
 
 jasmine.DEFAULT_TIMEOUT_INTERVAL = 20000;
 
-const ticksService = new TicksService(new LiveApi({ websocket, appId: 1169 }));
+const ticksService = new TicksService(new LiveApi({ websocket, appId: getAppIdFallback() }));
```

Comment:
Can we have a function that returns a new api instance with the default appid and websocket? this seems like a common use case.

Reply:
Done